sdk/core/Azure.Core/tests/CloudEventTests.cs

```diff
@@ -0,0 +1,316 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Threading;
+using Azure.Core.Serialization;
+using Azure.Messaging;
+using NUnit.Framework;
+
+namespace Azure.Core.Tests
+{
+    public class CloudEventTests
+    {
+        [Test]
+        public void ConstructorValidatesRequiredParameters()
+        {
+            Assert.That(
```

Comment:
If one of these assertions fails, how do you know which one?    Maybe consider adding a message at the end of the assert, an additional condition to validate the parameter name in the exception,  or breaking into two tests?

Reply:
It would be known based on the line number where the test failed.

Reply:
.... which isn't clear from just looking at the results.    A message, additional check, or splitting the test would be.   😄

Reply:
But you would likely need to look at the code anyways before fixing or creating an issue. I personally don't see the benefit, but I respect your opinion.